x-pack/plugins/ml/public/application/util/chart_utils.js

```diff
@@ -224,17 +224,29 @@ export async function getExploreSeriesLink(mlLocator, series, timeRange) {
 }
 
 export function showMultiBucketAnomalyMarker(point) {
-  // TODO - test threshold with real use cases
-  return (
-    point.multiBucketImpact !== undefined && point.multiBucketImpact >= MULTI_BUCKET_IMPACT.MEDIUM
-  );
+  return point.isMultiBucketAnomaly === true;
 }
 
 export function showMultiBucketAnomalyTooltip(point) {
-  // TODO - test threshold with real use cases
-  return (
-    point.multiBucketImpact !== undefined && point.multiBucketImpact >= MULTI_BUCKET_IMPACT.LOW
-  );
+  return point.isMultiBucketAnomaly === true;
+}
+
+export function getMultiBucketImpactTooltipValue(point) {
+  const numFilledSquares =
+    point.multiBucketImpact !== undefined
+      ? getAnomalyScoreExplanationImpactValue(point.multiBucketImpact)
+      : 0;
+  const numHollowSquares = 5 - numFilledSquares;
+
+  let tooltip = '';
+  for (let i = 0; i < numFilledSquares; i++) {
+    tooltip += '\u25A0 '; // Unicode filled square
+  }
+  for (let i = 0; i < numHollowSquares; i++) {
+    tooltip += '\u25A1 '; // Unicode hollow square
+  }
+
+  return tooltip;
```

Comment:
there are probably 100 different ways to do this, but here's a suggestion tat uses `fill`:

```ts
return new Array(5).fill('\u25A0 ', 0, numFilledSquares).fill('\u25A1 ', numFilledSquares).join('')
```